Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/TestPages/TestPages.cs

```diff
@@ -612,7 +612,7 @@ public ContentPage AddTopTab(string title, string icon = null)
 
 		public void AddTopTab(ContentPage page, string title = null, string icon = null)
 		{
-			if (Items.Count == 0)
+			if(Items.Count == 0 || Items[0].Items.Count == 0)
```

Comment:
Is this change needed?

If we're wanting to act on this being true *Items[0].Items.Count == 0* then we would add a ShellSection to Items[0] not call AddContentPage

But when I run your test code this condition is never true so I'm assuming we can just revert this change?

Reply:
I've changed this to use Items[0].Clear() and, after that, call "AddTopTab".
But, now I'm calling Items.Clear() too, so we can remove this...